org.eclipse.xtext.xtext.ui.examples/projects/arithmetics/org.eclipse.xtext.example.arithmetics/src-gen/org/eclipse/xtext/example/arithmetics/arithmetics/impl/AbstractDefinitionImpl.java

```diff
@@ -174,7 +174,7 @@ public String toString()
   {
     if (eIsProxy()) return super.toString();
 
-    StringBuilder result = new StringBuilder(super.toString());
+    StringBuffer result = new StringBuffer(super.toString());
```

Comment:
This looks like an old emf version to me.

Reply:
i dont know how this was generated before